modules/android/pages/working-with-vector-search.adoc

```diff
@@ -36,10 +36,10 @@ include::java:example$snippets/common/main/java/com/couchbase/codesnippets/Vecto
 
 First, initialize the `config` variable with the `VectorIndexConfiguration()` method with the following parameters:
 
-* The width or `dimensions` of the vector index is set to `300`.
+* The width or `dimensions` of the vector index is set to `3`.
 
-* The amount of `centroids` is set to `20`.
-This means that there will be 20 buckets with a single centroid each that gathers together similar vectors.
+* The amount of `centroids` is set to `100`.
+This means that there will be 100 buckets with a single centroid each that gathers together similar vectors.
```

Comment:
Is there somewhere else in the mobile docs that defines `bucket` in opposition to regular Couchbase Server buckets?

Reply:
"bucket" is a fairly common name in algorithms... but that's a good point from Richard 🤔

Reply:
I shall raise it with Iveta in the meantime, seems like a useful thing to discuss with Iveta.